presentations/en/2021-micro-acquisition-elevator-pitch.md

```diff
@@ -0,0 +1,103 @@
+<!--markdownlint-disable MD003 MD033 MD041 MD036 -->
+
+<style>
+.reveal section img { background:none; border:none; box-shadow:none; }
+</style>
+
+# Micro-Acquisition Pilot
+
+ESDC IT Strategy Team
+
+Note:
+
+- we are here today to talk to you about our procurement pilot and we will
+- but first we want to talk a little about what we believe
+
+---
+
+Trying to continuously improve while also keeping the lights on is hard
+
+Note:
+
+---
+
+Learning new things is essential to keep up with the fast pace of IT
+
+Note:
+
+(and we know that learning often takes a back seat to operational activities)
+
+---
+
+Sharing code across teams, branches and departments can free up valuable time
+
+Note:
+
+- reduce duplication
+- participate in a community
+- OSS code is more secure
+
+---
+
+IT contract dollars going to the same usual suspects reduces our exposure to new ideas
+
+Note:
+
+---
+
+There is a vast amount of IT talent out there for whom a $10K contract would mean a whole lot
+
+Note:
```

Comment:
University students, mono parental mom/dad needing an extra income to meet ends  (similar to values of ESDC), etc.